src/main/java/com/soundswapper/SoundSwapperPlugin.java

```diff
@@ -122,15 +106,15 @@ private synchronized void playCustomSound(String sound_name) {
 			clip.loop(0);
 		} catch (LineUnavailableException e) {
 			log.warn("Unable to play custom sound " + sound_name, e);
-			return;
 		}
 	}
 
 	private boolean tryLoadSound(String sound_name, File sound_file)
 	{
 		if (sound_file.exists())
 		{
-			try (InputStream fileStream = new BufferedInputStream(new FileInputStream(sound_file));
+
+			try (InputStream fileStream = Files.newInputStream(sound_file.toPath());
```

Comment:
The audio file doesn't play when written without the `new BufferedInputStream()`. I also tested the following, which did work:
```
try (InputStream fileStream = new BufferedInputStream(Files.newInputStream(sound_file.toPath()));
```